sgx/src/attestation_types/ti.rs

```diff
@@ -4,10 +4,11 @@
 //! The Target Info is used to identify the target enclave that will be able to cryptographically
 //! verify the REPORT structure returned by the EREPORT leaf. Must be 512-byte aligned.
 
+use crate::attestation_types::report;
 use crate::types::{attr::Attributes, misc::MiscSelect};
 
 /// Table 38-22
-#[derive(Debug)]
+#[derive(Default, Debug)]
```

Comment:
I don't think you need this. See below.

Reply:
I use the target_info with default values to call get_report.
https://github.com/enarx/enarx/pull/791/files#diff-f0d70ca796b4119abc3346153af29ce6R7

Reply:
Yes, of course. Sorry for the noise!